### ISSUE TITLE
Creation of new project is failing

### ISSUE THREAD

**Issue by james-k-mathew** [getgauge/gauge#1987]
**Describe the bug**
When trying to create a new Taiko-Gauge project, it is failing with the error
The specified program requires a newer version of Windows.. Please use a valid Gauge template URI or check network connection

**To Reproduce**
Create a new directory 
run the command
    gauge init js
Instead of project creation, getting the error message

Failed to initialize project. failed to unzip file C:\Users\uid\AppData\Local\Temp\gauge_temp1620711647188177900\js.zip: read C:\Users\uid\AppData\Local\Temp\gauge_temp1620711647188177900\js.zip: The specified program requires a newer version of Windows.. Please use a valid Gauge template URI or check network connection

Gauge version: 1.1.7
Commit Hash: 5d86b72

Plugins
-------
html-report (4.0.12)
js (2.3.15)
screenshot (0.0.1)

 - Windows 10 Pro    Version 20H2
 - IDE info, using VS Code IDE, but creating project from there fails without giving any error message

**Additional context**
I am not an administrator in the laptop. I use Avecto to install software in the laptop.

**Comment by zabil** [comment]
> I am not an administrator in the laptop. I use Avecto to install software in the laptop.

It might be related to as gauge installs plugins in the user directory. Can you try setting the environment variable [`GAUGE_HOME`](https://docs.gauge.org/faqs.html?os=linux&language=javascript&ide=vscode#what-is-gauge-home) to a folder where there are permissions?

**Comment by james-k-mathew** [comment]
I created a directory under user dir and set it as GAUGE_HOME. Still getting the same issue.


C:\Users\uid\NewGauge>echo %GAUGE_HOME%
C:\Users\uid\GaugeHome

C:\Users\uid\NewGauge>gauge init js
Initializing template from https://github.com/getgauge/template-js/releases/latest/download/js.zip
.
Error ----------------------------------

[Gauge]
Failed to initialize project. failed to unzip file C:\Users\uid\AppData\Local\Temp\gauge_temp1620720148089622600\js.zip: read C:\Users\uid\AppData\Local\Temp\gauge_temp1620720148089622600\js.zip: The specified program requires a newer version of Windows.. Please use a valid Gauge template URI or check network connection

Get Support ----------------------------
        Docs:          https://docs.gauge.org
        Bugs:          https://github.com/getgauge/gauge/issues
        Chat:          https://github.com/getgauge/gauge/discussions

Your Environment Information -----------
        windows, 1.1.7, 5d86b72
        Could not retrieve plugin information.

C:\Users\uid\NewGauge>

**Comment by zabil** [comment]
From the logs there does not seem to be an issue downloading the file. And the error seems to happen while unzipping. 

This specific error either happens due to admin privliges issue or maybe the zip command is incompatible. Can you try unzipping the file from the command line?

Alternatively you can just run 

```
git clone  https://github.com/getgauge/template-js 
cd template-js
npm install
```

or unzip https://github.com/getgauge/template-js/releases/latest/download/js.zip and start using the project structure.

However `npm` and node will need the right privileges to run too. Please check these alternatives.

**Comment by james-k-mathew** [comment]
After doing the git clone, cd template-js, npm install  The project got created in the template-js directory. When I tried to do a
    gauge run .\specs\example.spec
it was expecting the Plugin JS under the GAUGE_HOME directory. I removed the GAUGE_HOME entry and tried again. 
This time the tests related to example.spec ran successfully.

We will take this as alternate way to create projects for users who are not Administrators in their laptop.
